source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -48,10 +48,10 @@ namespace PlayFab
             WriteEventsResponse outResult;
             if (ValidateResult(outResult, container))
             {
-                const auto internalPtr = container.successCallback.get();
+                const auto& internalPtr = container.successCallback.get();
```

Comment:
>auto [](start = 22, length = 4)

it's not clear from the right-hand side what this type is.
recommend using the full type